packages/components/src/segmented-control/styles.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import { css } from 'emotion';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { CONFIG, COLORS, reduceMotion } from '../utils';
+
+export const SegmentedControl = css`
+	background: ${ COLORS.ui.background };
+	border: 1px solid;
+	border-color: ${ COLORS.ui.border };
+	border-radius: ${ CONFIG.controlBorderRadius };
+	display: inline-flex;
+	min-height: ${ CONFIG.controlHeight };
+	min-width: 0;
+	padding: 1px;
+	position: relative;
+	transition: all ${ CONFIG.transitionDurationFastest } linear;
+	${ reduceMotion( 'transition' ) }
+	&:hover {
+		border-color: ${ COLORS.ui.borderHover };
+	}
+
+	&:focus {
+		border-color: ${ COLORS.ui.borderFocus };
+		box-shadow: ${ CONFIG.controlBoxShadowFocus };
+		outline: none;
+		z-index: 1;
+	}
+`;
+
+export const block = css`
+	display: flex;
+	width: 100%;
+`;
+
+export const BackdropView = styled.div`
+	background: ${ COLORS.ui.background };
+	border: 1px solid ${ CONFIG.segmentedControlBackdropBorderColor };
+	border-radius: ${ CONFIG.controlBorderRadius };
+	box-shadow: ${ CONFIG.segmentedControlBackdropBoxShadow };
+	height: calc( ${ CONFIG.controlHeight } - 4px );
+	left: 0;
+	position: absolute;
+	transition: all ${ CONFIG.transitionDurationFast } ease;
+	${ reduceMotion( 'transition' ) }
+	z-index: 1;
+`;
+
+export const LabelView = styled.div`
+	display: inline-flex;
+	max-width: 100%;
+	min-width: 0;
+	position: relative;
+`;
+
+export const labelBlock = css`
+	flex: 1;
+`;
+
+export const buttonActive = css`
+	color: ${ COLORS.white } !important;
```

Comment:
@sarayourfriend do you know how to enforce these styles in higher priority? The important needs to go obviously 😄